korebuild-lock.txt

```diff
@@ -0,0 +1,2 @@
+version:2.1.0-preview1-15549
```

Comment:
What's this file do? Are we going to have to re-commit it every time a new KoreBuild pieces gets built?

Reply:
This file pins the version of KoreBuild so a clean repo doesn't just download 'latest' everytime you run build.cmd. It will get updated about once a week along with the rest of the versions in build\dependencies.props. Of course, you can also update it sooner manually if you need a newer version of KoreBuild.